Client/game_sa/C3DMarkerSA.h

```diff
@@ -1,50 +1,18 @@
 /*****************************************************************************
  *
- *  PROJECT:     Multi Theft Auto v1.0
+ *  PROJECT:     Multi Theft Auto
  *  LICENSE:     See LICENSE in the top level directory
  *  FILE:        game_sa/C3DMarkerSA.h
  *  PURPOSE:     Header file for 3D Marker entity class
  *
- *  Multi Theft Auto is available from http://www.multitheftauto.com/
+ *  Multi Theft Auto is available from https://www.multitheftauto.com/
  *
  *****************************************************************************/
 
 #pragma once
 
 #include <game/C3DMarker.h>
-#include <CMatrix_Pad.h>
-
-class C3DMarkerSAInterface
-{
-public:
-    CMatrix_Padded m_mat;                    // local space to world space transform // 0
-    DWORD          dwPad, dwPad2;            // not sure why we need these, it appears to be this way though (eAi)  // 64/68
-    RpClump*       m_pRwObject;              // 72
-    DWORD*         m_pMaterial;              // 76
-
-    WORD  m_nType;                  // 80
-    bool  m_bIsUsed;                // has this marker been allocated this frame?    // 82
-    DWORD m_nIdentifier;            // 84
-
-    DWORD rwColour;                    // 88
-    WORD  m_nPulsePeriod;              // 92
-    short m_nRotateRate;               // deg per frame (in either direction) // 94
-    DWORD m_nStartTime;                // 96
-    float m_fPulseFraction;            // 100
-    float m_fStdSize;                  // 104
-    float m_fSize;                     // 108
-    float m_fBrightness;               // 112
-    float m_fCameraRange;              // 116
-
-    CVector m_normal;            // Normal of the object point at             // 120
-    // the following variables remember the last time we read the heigh of the
-    // map. Using this we don't have to do this every frame and we can still have moving markers.
-    WORD    m_LastMapReadX, m_LastMapReadY;            // 132 / 134
-    float   m_LastMapReadResultZ;                      // 136
-    float   m_roofHeight;                              // 140
-    CVector m_lastPosition;                            // 144
-    DWORD   m_OnScreenTestTime;                        // time last screen check was done // 156
-};
+#include "interfaces/C3DMarkerSAInterface.h"
```

Comment:
The idea is interesting, but i think there were some sort of design choices involved. So we store SA interfaces and wrappers in the same file.
What do the other contributors think about that?

Reply:
@tederis @botder ?

Reply:
I think that generally the separation of MTA wrappers and SA internal structures is reasonable. I don't think that this can harm the existed code in any way.